README.md

```diff
@@ -20,7 +20,10 @@ of the Module Developer's Guide.
 | `onEresourceSelected` | func: (eresource) => {} | Callback fired when a user clicks an eresource | Yes |
 | `dataKey` | string | Optional `dataKey` passed to stripes/connect when creating the connected Eresources component. |  |
 | `renderTrigger` | func: ({ triggerId, onClick, buttonRef }) => {} | Optional render function for the button to open the Eresource search modal. The `onClick` prop should be called when the trigger is clicked (assuming it is a Button). The `buttonRef` ensures that the trigger button is brought back into focus once the lookup modal is closed| |
-| `showPackages` | boolean | Optional prop to only display the packages eresources and its necessary filters | |
+| `showPackages` | boolean | Optional prop to only display the packages eresources and its necessary filters | true |
```

Comment:
Are these `Required` `true` or `Default` `true`? I think there should be another column, right?